backend/src/main/java/com/woowacourse/gongseek/article/application/ArticleService.java

```diff
@@ -120,8 +105,8 @@ private Article getArticle(Long id) {
     }
 
     private List<String> getTagNames(Article article) {
-        List<ArticleTag> articleTags = articleTagRepository.findAllByArticleId(article.getId());
-        return articleTags.stream()
+        ArticleTags articleTags = article.getArticleTags();
```

Comment:
```suggestion
        List<ArticleTag> articleTags = article.getArticleTags().getArticleTags();
        return articleTags.stream()
                .map(articleTag -> articleTag.getTag().getName().getValue())
                .collect(Collectors.toList());
```

List로 뽑아서 사용하는건 어떤가요?

Reply:
수정했습니다!